utils/amdvlk_build_for_tag.py

```diff
@@ -36,26 +31,61 @@ def __init__(self):
         self.buildTag     = ''
         self.driverRoot   = ''
         self.diffTag      = 'v-2022.Q3.1'
-        self.validTags    = []
+        self.accessToken  = ''
 
-    def UpdateValidTags(self, repoPath):
-        repo = Github().get_repo(repoPath)
+    def IsBuildTagNewer(self):
+        #TODO: use a more flexible algorithm
+        return self.buildTag > self.diffTag
+
+    def SetupBuildTagInfo(self, inputTag):
+        ownerName = self.targetRepo.strip('/ ').split('/')[-1]
+        repoName = 'AMDVLK'
+        g = Github(self.accessToken) if self.accessToken else Github()
```

Comment:
Is the accessToken essential for this new remote tag require function?

BTW, if we checkout the repo entirely every time we run this python, why people will make mistake about tag?

Reply:
> Is the accessToken essential for this new remote tag require function?

Not a must, it is recommended to use access token to avoid potential Github's rate-limiting issue.

> 
> BTW, if we checkout the repo entirely every time we run this python, why people will make mistake about tag?

For test or on purpose, the specified build tag in local clone repo AMDVLK may be unavailable, so we should use remote tag.